admin/templates/admin/admin.html

```diff
@@ -81,10 +81,10 @@ <h2 class="card-title">Update Publication Terms</h2>
                             <textarea class="form-control" id="publicationTerms" name="publicationTerms" rows="10"
                                 style="width: 110%;" required>{{ publication_terms|safe }}</textarea>
                         </div>
-                        <button type="submit" class="btn btn-primary">Update Terms</button>
+                        <button type="submit" name="Update Terms" class="btn btn-primary">Update Terms</button>
                     </form>
                     <!-- Form to preview updated terms -->
-                    <button type="button" id="create-preview" class="btn btn-secondary">Preview Terms</button>
+                    <button type="button" id="create-preview" name="Preview Terms" class="btn btn-secondary">Preview Terms</button>
```

Comment:
This button already had an unique id for the UI tests: `create-preview`. We should use that instead.